src/main/java/com/voice/dbRegistration/dao/DevoteeInfoDao.java

```diff
@@ -9,6 +9,7 @@
 import com.voice.dbRegistration.model.DevoteeInfo;
 
 public interface DevoteeInfoDao extends JpaRepository<DevoteeInfo, String> {
+    @Query(value = "select * FROM DevoteeInfo d where d.connectedTo like '%guru%'")
```

Comment:
We don't need "d" here. We are not using any join here

Reply:
applied changes